File: velolabs_repos/lattis_android/app/src/main/java/com/lattis/ellipse/data/network/model/response/PrivateNetworkResponse.java
package com.lattis.ellipse.data.network.model.response;

import com.google.gson.annotations.SerializedName;

/**
 * Created by ssd3 on 5/8/17.
 */

public class PrivateNetworkResponse {

    @SerializedName("private_fleet_user_id")
    private int private_fleet_user_id;
    @SerializedName("user_id")
    private int user_id;
    @SerializedName("email")
    private String email;
    @SerializedName("fleet_id")
    private int fleet_id;
    @SerializedName("verified")
    private int verified;
    @SerializedName("fleet_name")
    private String fleet_name;
    @SerializedName("type")
    private String type;
    @SerializedName("logo")
    private String logo;

    public int getPrivate_fleet_user_id() {
        return private_fleet_user_id;
    }

    public int getUser_id() {
        return user_id;
    }

    public String getEmail() {
        return email;
    }

    public int getFleet_id() {
        return fleet_id;
    }

    public int getVerified() {
        return verified;
    }

    public String getFleet_name() {
        return fleet_name;
    }

    public String getType() {
        return type;
    }

    public String getLogo() {
        return logo;
    }




}

File: velolabs_repos/lattis_android_v2/data/src/main/java/com/lattis/data/entity/response/private_networks/PrivateNetworkResponse.kt
package com.lattis.data.entity.response.private_networks

import com.google.gson.annotations.SerializedName

/**
 * Created by ssd3 on 5/8/17.
 */

class PrivateNetworkResponse {

    @SerializedName("private_fleet_user_id")
    val private_fleet_user_id: Int = 0
    @SerializedName("user_id")
    val user_id: Int = 0
    @SerializedName("email")
    val email: String? = null
    @SerializedName("fleet_id")
    val fleet_id: Int = 0
    @SerializedName("verified")
    val verified: Int = 0
    @SerializedName("fleet_name")
    val fleet_name: String? = null
    @SerializedName("type")
    val type: String? = null
    @SerializedName("logo")
    val logo: String? = null
    @SerializedName("address")
    val address: Address? = null

    class Address{
        var address_id:Int? =null
        var city: String?=null
        var address1: String?=null
        var address2: String?=null
        var state: String?=null
        var postal_code: String?=null
        var country: String?=null
        var type: String?=null
        var type_id: Int?=null
    }


}
